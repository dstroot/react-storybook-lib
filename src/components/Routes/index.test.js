import React from "react";
import renderer from "react-test-renderer";

// import component to test
import Routes from "../Routes";

// Test setup
const Home = () => <div>Home</div>;
const About = () => <div>Home</div>;
const testData = [
  { path: "/", name: "Home", component: Home },
  { path: "/about", name: "About", component: About }
];

describe("Routes", () => {
  it("it should render", () => {
    const component = renderer.create(<Routes data={testData} />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
