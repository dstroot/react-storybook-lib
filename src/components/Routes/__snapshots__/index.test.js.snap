// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Routes it should render 1`] = `
<div
  role="group"
  style={
    Object {
      "outline": "none",
    }
  }
  tabIndex="-1"
>
  <div>
    Home
  </div>
</div>
`;
